plugins/foreman_azure/2.1.x/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: plugin
+title: Foreman AzureRm 2.1.x Manual
+version: 2.1.x
```

Comment:
I think you'll need to also add this version to the plugin index

Reply:
done.